module-apps/apps-common/ApplicationCommon.cpp

```diff
@@ -467,13 +467,13 @@ namespace app
 
     sys::MessagePointer ApplicationCommon::handleMinuteUpdated(sys::Message *msgl)
     {
-        if (state == State::ACTIVE_FORGROUND && getCurrentWindow()->updateTime()) {
-
+        if (state == State::ACTIVE_FORGROUND) {
+            auto requestedRefreshMode = getCurrentWindow()->updateTime();
             if (isOnPhoneLockWindow()) {
                 updateStatusBarOnPhoneLockWindow();
             }
```

Comment:
didn't we agree that it can be moved to the `PhoneLockWindow` and both these methods can be removed from the ApplicationCommon? :) It's not much, but when we can remove unrelated code from "common" parts - it's worth it ;)

Reply:
isOnPhoneLockWindow() is used in multiple different places, so I left this method, but updateStatus... content was moved to PhoneLockedWindow as you suggested ;)

Reply:
tx, I'll see if it could be possibly removed in other places too :)